tests/testTriggers.go

```diff
@@ -5690,6 +5692,107 @@ func TriggerAsyncDR(contexts *[]*scheduler.Context, recordChan *chan *EventRecor
 	updateMetrics(*event)
 }
 
+// TriggerAsyncDRVolumeOnly triggers Async DR
+func TriggerAsyncDRVolumeOnly(contexts *[]*scheduler.Context, recordChan *chan *EventRecord) {
+	logrus.Infof("Async DR triggered at: %v", time.Now())
```

Comment:
Please pull the latest changes and use dash anf log for logging

Reply:
Done, used dash.